stable/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -78,25 +77,29 @@ spec:
           protocol: TCP
         {{- end }}
         env:
+          - name: DD_HEALTH_PORT
+            value: {{ .Values.clusterAgent.healthPort | default 8080 | quote }}
           - name: DD_API_KEY
             valueFrom:
               secretKeyRef:
                 name: {{ template "datadog.apiSecretName" . }}
                 key: api-key
-          {{- if .Values.clusterAgent.metricsProvider.enabled }}
-          - name: DD_EXTERNAL_METRICS_PROVIDER_ENABLED
-            value: {{ .Values.clusterAgent.metricsProvider.enabled | quote }}
+                optional: true
           - name: DD_APP_KEY
             valueFrom:
               secretKeyRef:
                 name: {{ template "datadog.appKeySecretName" . }}
                 key: app-key
```

Comment:
the key "app-key" and "api-key" is hardcoded like this since a wild and we didn't received any issue about it.
So I prefer to not introduce a new parameter if everyone is ok like this.
WDYT?

Reply:
That's fine. I only suggested it since this was a major change to the chart.

Reply:
can be added after this it will only introduce a new parameters